content/200-concepts/100-components/250-preview-features/index.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title: 'Preview features'
+metaTitle: 'Preview features (Reference)'
+metaDescription: 'A list of Prisma features currently in Preview.'
+hiddenPage: false
+---
+
+<TopBlock>
+
+Some Prisma features are released as [Previews](../../../about/releases#preview). [Share your feedback on all Preview features on GitHub](https://github.com/prisma/prisma/issues/3108).
+
+</TopBlock>
+
+## Enabling a Prisma CLI preview feature
+
+To use a Prisma CLI Preview feature, append the `--preview-feature` flag each time you run the command:
+
+```terminal
+npx prisma db push --preview-feature
+```
+
+## Enabling a Prisma Client preview feature
+
+To enable a Prisma Client Preview feature:
+
+1. Add the Preview feature flag to the `client` block:
+
+   ```prisma
+    generator client {
+    provider        = "prisma-client-js"
+    previewFeatures = ["orderByRelation"]
+    } 
+    ```  
+
+2. Re-generate the client:
+
+   ```terminal
+   npx prisma generate
+   ```
+
+3. If you are using Visual Studio Code and the Preview feature is not available in your `.ts` file after generating the client, run the **TypeScript: Restart TS server** command.
```

Comment:
This should probably list to the new files we have that list CLI and Client preview features individually (also part of this PR)